docs/ko/docs/tutorial/middleware.md

```diff
@@ -0,0 +1,61 @@
+# 미들웨어
+
+미들웨어를 **FastAPI** 애플리케이션에 추가할 수 있습니다.
```

Comment:
```suggestion
미들웨어를 **FastAPI** 응용 프로그램에 추가할 수 있습니다.
```

If you check [파이썬 한국어 번역](https://www.flowdas.com/pages/python-docs-ko.html) and search word, `application`, it is translated as `응용 프로그램` and because we follows the rules of that guide (check of it on #2017 ).